consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/impl/JudgementWarrantService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.moj.sdt.cmc.consumers.client.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.moj.sdt.cmc.consumers.api.CMCApi;
+import uk.gov.moj.sdt.cmc.consumers.api.IJudgementWarrantService;
+import uk.gov.moj.sdt.cmc.consumers.request.JudgementWarrantRequest;
+import uk.gov.moj.sdt.cmc.consumers.response.JudgementWarrantResponse;
+
+@Service("JudgementWarrantService")
+public class JudgementWarrantService implements IJudgementWarrantService {
+
+    private CMCApi cmcApi;
+
+    @Autowired
+    public JudgementWarrantService(CMCApi cmcApi) {
+        this.cmcApi = cmcApi;
+    }
+
+    @Override
+    public JudgementWarrantResponse judgementWarrantRequest(String authorization,
+                                                            String serviceAuthorization,
```

Comment:
These two parameters aren't on the corresponding method on the other service classes (e.g. JudgementRequestService.requestJudgment()) - are they needed?

Reply:
Better to have them, service authorisation is needed to communicate with CMC later part